<launch>

  <arg name="bag_path" default="" /> 
  <arg name="save_dir" default="" />
  <arg name="publish_framerate" default="60.0" />
  <arg name="calibration_path" />
  <arg name="node_name" default="reconstruction_node" />
  
  <!-- pure event reconstruction -->
  <node name="$(arg node_name)" pkg="pure_event_reconstruction" type="reconstruction_node" output="screen" args="--v=3">
    <param name="working_dir" value="$(env PWD)"/>
    <param name="bag_path" type="str" value="$(arg bag_path)"/>
    <param name="save_dir" type="str" value="$(arg save_dir)"/>
	<param name="publish_framerate" type="double" value="$(arg publish_framerate)"/>
    <param name="calibration_path" type="str" value="$(arg calibration_path)" />
    <param name="node_name" type="str" value="$(arg node_name)" />
	
    <remap from="events" to="dvs/events" />
  </node>
  
</launch>
